prisma/seeds/roles.ts

```diff
@@ -7,19 +7,41 @@ export async function seedRoles() {
   await prisma.roles.createMany({
     data: [
       {
-        name: 'Admin',
-        description: 'Responsible for managing a resource'
+        id: "665fdcce-8795-4cec-8c91-0ff1c47e5284",
+        name: "Admin",
+        description: "Responsible for managing a resource",
       },
       {
-        name: 'Train Operator',
-        description: 'Responsible for operating the train'
+        id: "faf787c0-6b97-4dae-84e4-ca947c790bf2",
+        name: "Train Operator",
+        description: "Responsible for operating the train",
       },
       {
-        name: 'Ticket Booth Attendant',
-        description: 'Responsible for attending the box office'
-      }
-    ]
+        id: "b7df9f50-0b5c-45f9-ae54-5c37de8c97ae",
+        name: "Ticket Booth Attendant",
+        description: "Responsible for attending the box office",
+      },
+      {
+        id: "477b3243-8a2e-484f-baf2-18bf7f08aa31",
+        name: "Maintains technical services",
+        description: "Responsible for attending technical services",
+      },
+      {
+        id: "8659bc51-fbef-42e7-a0c5-39fe219d2e27",
+        name: "Passenger",
+        description: "Passanger role",
```

Comment:
Melhorar descrição:

`Uses the subway for transportation, plans journeys, buys tickets, and receives service updates.`